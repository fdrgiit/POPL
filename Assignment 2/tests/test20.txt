s a b c #
A B C D #
s -> a A #
a -> b B #
b -> c C #
c -> s D #
s -> #
a -> #
b -> #
c -> #
##

