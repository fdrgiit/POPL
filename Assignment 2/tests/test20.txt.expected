FIRST(s) = { #, A, B, C, D }
FIRST(a) = { #, A, B, C, D }
FIRST(b) = { #, A, B, C, D }
FIRST(c) = { #, A, B, C, D }
FOLLOW(s) = { $, D }
FOLLOW(a) = { A }
FOLLOW(b) = { B }
FOLLOW(c) = { C }
